src/web/components/LayoutSlotBottom.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { css } from 'emotion';
+import { useApi } from '@root/src/web/components/lib/api';
+
+const wrapperMargins = css`
+    margin: 18px 0;
+`;
+
+export const LayoutSlotBottom = () => {
```

Comment:
What about something a bit more specific? As we will definitely have other '*Bottom' slots in the future/ones that live below this one in fact. Something like:

    ArticleEndSlot

perhaps?

Reply:
To be honest, I'm completely fine with `LayoutSlotBottom` for now - it describes what this is. I just wanted to plant the seed that it might need reviewing later as we grow. I defo don't want to slow down getting this out to prod on naming niggles.

Reply:
We haven't agreed on any slot names yet, so they're bound to change no matter what. For now I wanted something obvious and descriptive, but I expect this to change soon so not sure it's worth deviating much more from this...